library/std/src/process.rs

```diff
@@ -2294,6 +2294,25 @@ impl Child {
 /// considered undesirable. Note that returning from `main` also calls `exit`, so making `exit` an
 /// unsafe operation is not an option.)
 ///
+/// ## Safe interop with C code
```

Comment:
```suggestion
/// ## Safety
```
I think usually, this section is called "Safety".

Reply:
I don't think we have any safe function with a "safety" section.

Reply:
We used to have one for the `std::env::{set_var, remove_var}` functions, even before they were marked as `unsafe`. I think this is a similar situation.

Reply:
We decided the situation is not quite similar, since we won't make `exit` unsafe.